STM32F405RGTx/Core/Inc/state_machine.hpp

```diff
@@ -0,0 +1,104 @@
+/* state_machine.h
+ *
+ *    Created on: Jul. 11, 2021
+ *            Author: Kanishk Kashyap
+ */
+#pragma once
+
+#include "can.h"
+#include "cmsis_os.h"
+#include "can.h"
+#include "util.hpp"
+
+#ifndef _STATE_MACHINE_H_
```

Comment:
Replace these `#ifndef`'s and `#define`'s with `#pragma once` pls. Same with all the other files that have this.